src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -383,8 +401,23 @@ const EditHomepage = ({ match }) => {
             textCardItemErrors.push(
               _.map(cards, () => getErrorValues(TEXTCARDS_ITEM_SECTION))
             )
+          }
+
+          // TODO: seems to load an existing state incorrectly into the errors object
```

Comment:
I faced this error, i think it might have something to do with the useEffect updating errors. My concern is not mainly is the root problem found and solved, but more of is this a concern of is this to impt for production? ie are we sure that the error states are correctly shown to users?

Reply:
Oops, I fixed this already. Was updated in the child PR. Will remove this comment